### ISSUE TITLE
btoa is available while remote debugging on, undefined while remote debugging off 

### ISSUE THREAD

**Issue by manV** [facebook/react-native#10248]
### Issue Description

I have developed an app entirely with remote debugging is on and used `btoa` global function a lot, but as soon as I tried running without debugger it started failing saying that `btoa` is undefined.

### Steps to Reproduce / Code Snippets

Use `btoa` function and run while remote debugging on, and then try running app with remote debugging off.

#### Expected Results

`btoa` should be undefined when remote debugging is on too.

### Additional Information

* React Native version: 0.34.1
* Platform(s) (iOS, Android, or both?): Android
* Operating System (macOS, Linux, or Windows?): Ubuntu 16.04

**Comment by THIAGOWESLLEY2** [comment]
Thiago.wesley.oliveira@hotmail.com

Em 05/10/2016 06:56, Manan Vaghasiya <notifications@github.com> escreveu:
Issue Description

I have developed an app entirely with remote debugging is on and used btoa global function a lot, but as soon as I tried running without debugger it started failing saying that btoa is undefined.

Steps to Reproduce / Code Snippets

Use btoa function and run while remote debugging on, and then try running app with remote debugging off.

Expected Results

btoa should be undefined when remote debugging is on too.

Additional Information

  *   React Native version: 0.34.1
  *   Platform(s) (iOS, Android, or both?): Android
  *   Operating System (macOS, Linux, or Windows?): Ubuntu 16.04
